sway-lib-std/src/tx.sw

```diff
@@ -205,6 +207,27 @@ pub fn tx_input_coin_owner(input_ptr: u32) -> Address {
     owner_addr
 }
 
+////////////////////////////////////////
+// Inputs > Predicate
+////////////////////////////////////////
+
+pub fn tx_predicate_data_start_offset() -> u64 {
+    let is = instrs_start();
+    let predicate_length_ptr = is - 16;
+    let predicate_code_length = asm(r1, r2: predicate_length_ptr) {
+        lw r1 r2 i0;
+        r1: u64
+    };
+
+    let predicate_data_ptr = is + predicate_code_length;
```

Comment:
This should be 
```suggestion
    let predicate_data_ptr = is + predicate_code_length * 4;
```

The fact that it's not means there's a bug in the SDKs where they're not dividing the lengths by 4 to get the number of instructions instead of the number of bytes, as described in the specs.

Reply:
Only (1). Code length is in instructions (each instruction is 4 bytes), data length is in bytes.

Reply:
So does this still need to be resolved somewhere?  Is there an open issue to fix the code length thing?